### PR TITLE
Add a spec for mailchimp list jbuilder view

### PR DESCRIPTION
**Pull request by wwahammy**
We didn't have an actual spec for this jbuilder view so it made sense to add one quick.

**NOTE: DO NOT discuss internal CommitChange information in your PR; this PR will be public.
Link back to the issue in the Tix repo when you need to do that.**

### CODE REVIEW

File: spec/views/mailchimp/list.json.jbuilder_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+require 'rails_helper'
+
+RSpec.describe '/mailchimp/list.json.jbuilder', type: :view do
+
+	
+  describe 'supporter with no active recurring donations' do 
+		
+    subject(:json) do
+      view.lookup_context.prefixes = view.lookup_context.prefixes.drop(1)
+      assign(:supporter, create(:supporter, name: "Penelope Schultz"))
```
Comment (camckin10) [inline spec/views/mailchimp/list.json.jbuilder_spec.rb:14]: 😄  at the test name.

File: spec/views/mailchimp/list.json.jbuilder_spec.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      render
+      rendered
+    end
+
+
+    it {
+			is_expected.to include_json(
+				email_address: Supporter.first.email,
```
Comment (camckin10) [inline spec/views/mailchimp/list.json.jbuilder_spec.rb:22]: The other PR that you requested review for also used 'Supporter' or 'supporter.' 
Is this specifically from the Mailchimp documentation?

File: spec/views/mailchimp/list.json.jbuilder_spec.rb (hunk 1 above)
Comment (wwahammy) [reply spec/views/mailchimp/list.json.jbuilder_spec.rb:14]: She often shows up in the code base :smile:

File: spec/views/mailchimp/list.json.jbuilder_spec.rb (hunk 2 above)
Comment (wwahammy) [reply spec/views/mailchimp/list.json.jbuilder_spec.rb:22]: No, `Supporter` is a class in our code.

**Review by camckin10 — APPROVED** [review]
lgtm.

**Pull request merged**